### ISSUE TITLE
[Deep Fear] Missing legs.

### ISSUE THREAD

**Issue by BenjaminSiskoo** [devmiyax/yabause#389]
UoYabause 0.5.0

![deepfear](https://user-images.githubusercontent.com/12481242/27759961-857f12f6-5e3c-11e7-960b-18ac39d51987.JPG)

**Comment by paul-met** [comment]
Use HLE BIOS to fix it.

![01](https://user-images.githubusercontent.com/13948032/27764682-b909b232-5ea7-11e7-930b-5fe4d1d1602a.png)

**Comment by BenjaminSiskoo** [comment]
Yup, anyway, I guess the focus is on the real bios (and not the HLE). Especially for android users...

**Comment by paul-met** [comment]
You are welcome.
In addition, in order for the picture not to be cut off from the top of the screen, you need to select "fixed aspect ratio 4:3" in the video settings. But at the same time, unfortunately, the pixel aspect ratio will become impossible.

**Comment by BenjaminSiskoo** [comment]
:)

I completely forgot this setting (never thought to use it). Thanks a lot again ^^

**Comment by BenjaminSiskoo** [comment]
UoYabause 0.5.2 same problem

**Comment by BenjaminSiskoo** [comment]
UoYabause 0.5.2 commit 4105 : same problem

**Comment by BenjaminSiskoo** [comment]
Same problem with Yaba Sanshiro 1.3.0 RC

**Comment by BenjaminSiskoo** [comment]
Same problem with Yaba Sanshiro 2.2.0

**Comment by BenjaminSiskoo** [comment]
Same problem with Yaba 3.2 b2

**Comment by BenjaminSiskoo** [comment]
Same problem with Yaba 3.2.4
